tests/test_pfx.c

```diff
@@ -330,18 +330,25 @@ static void pfx_table_test(void)
 	printf("pfx_table_test successful\n");
 }
 
+static void create_ip4_pfx_record(struct pfx_record *pfx, uint32_t asn,
+			       const char *ip, uint8_t min_mask_len,
```

Comment:
indention looks off? not aligned with parentheses above?

Reply:
but maybe my browser is to blame here ...

Reply:
indent is off on my screen too